src/components/buttons/search.ts

```diff
@@ -39,27 +61,26 @@ export default {
 			}
 			else {
 				msg = 'MUSIC.QUEUE.TRACK_ADDED.MULTIPLE.DEFAULT';
-				extras = [resolvedTracks.length, await getGuildLocaleString(interaction.guildId, 'MISC.YOUR_SEARCH'), ''] ;
+				extras = [resolvedTracks.length.toString(), await getGuildLocaleString(interaction.guildId, 'MISC.YOUR_SEARCH'), ''] ;
 			}
 			if (!player?.connected) {
 				player = interaction.client.music.createPlayer(interaction.guildId);
 				player.handler = new PlayerHandler(interaction.client, player);
-				player.queue.channel = interaction.channel;
+				player.queue.channel = <TextChannel | VoiceChannel> interaction.channel;
```

Comment:
Based on usage, this is a TextChannel
```suggestion
				player.queue.channel = <TextChannel> interaction.channel;
```

Reply:
could be a voicechannel due to text in voice